instance/models/openedx_appserver.py

```diff
@@ -163,7 +163,7 @@ class Meta:
         null=True,
         blank=True,
         default=default_setting('OPENSTACK_SANDBOX_BASE_IMAGE'),
-        help_text='JSON openstack base image selector, e.g. {"name": "xenial-16.04-unmodified"}'
+        help_text='JSON openstack base image selector, e.g. {"name": "focal-20.04-unmodified"}'
```

Comment:
@0x29a, since this change triggers an additional migration, can we just leave it as-is. It is just the help text. Same for the 2 other similar changes.

Reply:
@lgp171188, not only this change triggers migrations. Note that the default value for this field is calculated lazily basing on settings. I included these migrations to update default values. During deployment, we'll have to check that `settings.OPENSTACK_SANDBOX_BASE_IMAGE` (and other settings, I mentioned all of them in testing instructions) have correct values.

Reply:
@0x29a, defining default values for a model based on a setting is a very bad practice. A callable should have been used instead :( But let's not worry about this now.